py/desispec/scripts/proc.py

```diff
@@ -694,6 +694,15 @@ def main(args=None, comm=None):
                 cmd += ' -o {}'.format(framefile)
                 cmd += ' --psferr 0.1'
 
+                if args.gpuspecter:
+                    cmd += ' --gpu-specter'
+                    #- default for CPU is nsubbundles=6 but gpu_specter only allows 1, 5, or 25
+                    cmd += ' --nsubbundles 5'
+                    cmd += ' --mpi'
```

Comment:
I'm a little surprised that the `--mpi` option appears necessary for the gpu_specter branch but not the old-specter branch, even though it is called with MPI parallelism as well later:
```
extract_args = desispec.scripts.extract.parse(cmdargs)

if args.gpuextract:
    desispec.scripts.extract.main_gpu_specter(extract_args, coordinator=coordinator)
elif args.gpuspecter:
    desispec.scripts.extract.main_gpu_specter(extract_args, comm=comm_extract)
else:
    desispec.scripts.extract.main_mpi(extract_args, comm=comm_extract)
```
i.e. what would happen if `main_gpu_specter` was called with a `comm` or `coordinator` but with `args.mpi = False`?

Reply:
I recall adding `--mpi` there so that it would show up when the command is logged. I don't think it is necessary.

Here is the logic in `main_gpu_specter`:

https://github.com/desihub/desispec/blob/fce0cf0fcf7c51ef0a37a918828e55f728a8545e/py/desispec/scripts/extract.py#L215-L230

Reply:
Leaving this in so that it is logged for gpu_specter.